### PR TITLE
Task: Add Pimcore 11 and Symfony 6 support, re-activate 11.x-dev tests

### LINKED ISSUE #185: Prepare for Symfony 6 upgrade
### Feature description

Related to https://github.com/pimcore/pimcore/issues/13242

and reactivate 11.x-dev tests (phpstan)

### PR DESCRIPTION
**Pull request by kingjia90**
Resolves https://github.com/pimcore/advanced-object-search/issues/185
Related https://github.com/pimcore/advanced-object-search/issues/193

### CODE REVIEW

File: .github/workflows/static-analysis.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
                 include:
                     - { php-version: "8.0", dependencies: "lowest", pimcore_version: "", phpstan_args: "", experimental: false }
                     - { php-version: "8.1", dependencies: "highest", pimcore_version: "", phpstan_args: "", experimental: false }
-                    #- { php-version: "8.1", dependencies: "highest", pimcore_version: "11.x-dev", phpstan_args: "", experimental: true }
+                    - { php-version: "8.1", dependencies: "highest", pimcore_version: "11.x-dev", phpstan_args: "", experimental: true }
```
Comment (fashxp) [inline .github/workflows/static-analysis.yml:25]: will require some changes further down too ... 
```
composer require --no-update pimcore/pimcore:"${PIMCORE_VERSION} as 10.99.9"
```
--> should say `11.99.9` I guess

**Review by fashxp — DISMISSED** [review]

**Comment by dvesh3** [comment]
Requires #176 for 11.x-dev tests to pass.

**Comment by fashxp** [comment]
@dvesh3 we could already change dependency from `ongr/elasticsearch-dsl` to `handcraftedinthealps/elasticsearch-dsl` ... then it should work also with lower versions of `elasticsearch-dsl` and symfony 6.

File: composer.json [hunk 2]
```diff
@@ -7,15 +7,15 @@
   },
   "require": {
     "ongr/elasticsearch-dsl": "^7.0",
```
Comment (dvesh3) [inline composer.json]:
```suggestion
    "handcraftedinthealps/elasticsearch-dsl": "^7.0",
```

File: composer.json [hunk 3]
```diff
@@ -6,16 +6,16 @@
...
-    "symfony/config": "^5.2.0",
-    "symfony/console": "^5.2.0",
-    "symfony/dependency-injection": "^5.2.0",
-    "symfony/event-dispatcher": "^5.2.0",
-    "symfony/http-foundation": "^5.2.0",
-    "symfony/http-kernel": "^5.2.0",
-    "symfony/messenger": "^5.2.0",
-    "symfony/routing": "^5.2.0"
+    "handcraftedinthealps/elasticsearch-dsl": "^7.0",
+    "pimcore/pimcore": "^10.5 || ^11",
```
Comment (dvesh3) [inline composer.json]:
```suggestion
    "pimcore/pimcore": "^10.6 || ^11.0",
```

**Pull request merged**